yt_dlp/extractor/rokfin.py

```diff
@@ -254,3 +324,85 @@ def _real_extract(self, url):
         return self.playlist_result(
             self._entries(channel_id, channel_name, self._TABS[tab]),
             f'{channel_id}-{tab}', f'{channel_name} - {tab.title()}', str_or_none(channel_info.get('description')))
+
+
+class RokfinSearchIE(SearchInfoExtractor):
+    IE_NAME = 'rokfin:search'
+    IE_DESC = 'Rokfin Search'
+    _SEARCH_KEY = 'rkfnsearch'
+    _TYPES = {
+        'video': (('id', 'raw'), 'post'),
+        'audio': (('id', 'raw'), 'post'),
+        'stream': (("content_id", 'raw'), 'stream'),
+        'dead_stream': (('content_id', 'raw'), 'stream'),
+        'stack': (('content_id', 'raw'), 'stack'),
+    }
+    _TESTS = [{
+        'url': 'rkfnsearch5:\"zelenko\"',  # The quotes are intentional.
+        'playlist_count': 5,
+        'info_dict': {
+            'id': '\"zelenko\"',
+            'title': '\"zelenko\"',
+        }
+    }]
```

Comment:
if this is meant to be the equivalent of typing `"zelenko"` in the site, it should be
```suggestion
    _TESTS = [{
        'url': 'rkfnsearch5:"zelenko"',
        'playlist_count': 5,
        'info_dict': {
            'id': '"zelenko"',
            'title': '"zelenko"',
        }
    }]
```